utilities/offerUtils/offerUtils.go

```diff
@@ -88,12 +90,14 @@ func UpdateEnvironment(offer *mesos.Offer) {
 	var host = offer.GetHostname()
 	// If this host is not present in the set of hosts.
 	if _, ok := constants.Hosts[host]; !ok {
-		log.Printf("New host detected. Adding host [%s]", host)
+		elekLog.ElektronLog.Log(elekLogTypes.CONSOLE, log.InfoLevel,
+			log.Fields{"Adding host": fmt.Sprintf("%s", host)}, "New host detected")
```

Comment:
As `host` is a string, you don't need to use `fmt.Sprintf`.

Reply:
Done